routing/index_graph.hpp

```diff
@@ -77,16 +119,154 @@ class IndexGraph final
   {
     uint32_t const jointsSize = ReadPrimitiveFromSource<uint32_t>(src);
     m_roadIndex.Deserialize(src);
-    m_jointIndex.Build(m_roadIndex, jointsSize);
+    Build(jointsSize);
+  }
+
+  Joint::Id GetJointIdForTesting(RoadPoint const & ftp) const
+  {
+    return m_roadIndex.GetJointId(ftp);
+  }
+
+  /// \brief Disable all edges between |from| and |to| if they are different and adjacent.
+  /// \note Despit the fact that |from| and |to| could be connected with several edges
+  /// it's a rare case. In most cases |from| and |to| are connected with only one edge
```

Comment:
So, does your code work properly if there're multiple edges between from and to? And if your code works properly, you need to say about it in the comment.